package.json

```
@@ -16,7 +16,7 @@
     "bugs": "https://github.com/ashtuchkin/iconv-lite/issues",
     "repository": {
         "type": "git",
-        "url": "git://github.com/ashtuchkin/iconv-lite.git"
+        "url": "https://github.com/ashtuchkin/iconv-lite.git"
```

This package is going to be moved to pillarjs, so I’ll go ahead and update the metadata now.

```suggestion
        "url": "https://github.com/pillarjs/iconv-lite.git"
```